File: ZKUIComponents/src/main/java/com.zkteam.ui.components/activity/SettingActivity.java
package com.zkteam.ui.components.activity;

import android.os.Bundle;
import android.view.View;
import android.widget.TextView;
import androidx.appcompat.app.AppCompatActivity;
import com.blankj.utilcode.util.ActivityUtils;
import com.zkteam.ui.components.R;

public class SettingActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_setting);
        findViewById(R.id.cons_about).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                ActivityUtils.startActivity(AboutActivity.class);
            }
        });

        TextView tvTitle = findViewById(R.id.tv_title);
        tvTitle.setText("设置");
    }
}

File: ZKUIComponents/src/main/java/com.zkteam.ui.components/activity/SettingActivity.kt
package com.zkteam.ui.components.activity

import android.os.Bundle
import android.view.View
import com.blankj.utilcode.util.ActivityUtils
import com.zkteam.sdk.base.ZKBaseActivity
import com.zkteam.ui.components.R
import kotlinx.android.synthetic.main.activity_setting.*
import kotlinx.android.synthetic.main.widget_toolbar.*

class SettingActivity: ZKBaseActivity() {
    override fun getLayoutId(): Int {
        return R.layout.activity_setting
    }

    override fun initData(bundle: Bundle?) {
        //function
    }

    override fun initLifecycleObserve() {
        //function
    }

    override fun initListener() {
        cons_about.setOnClickListener {
            ActivityUtils.startActivity(AboutActivity::class.java)
        }
    }

    override fun initViews(contentView: View) {
        tv_title.text = "设置"
    }

    override fun onDebouncingClick(view: View) {
        //function
    }
}